docs_app/src/app/app.component.ts

```diff
@@ -159,7 +159,7 @@ export class AppComponent implements OnInit {
     ).subscribe(([versionInfo, versions]) => {
       this.docVersions = [...versions];
 
-      // Find the current version - eithers title matches the current deployment mode
+      // Find the current version - either title matches the current deployment mode
```

Comment:
There was a risk that it was `either's` but I couldn't find a second thing, so I'm somewhat confident about this.